package.json

```diff
@@ -76,7 +76,7 @@
     "react-native-iphone-x-helper": "1.3.1",
     "react-native-lightbox-v2": "0.9.0",
     "react-native-parsed-text": "0.0.22",
-    "react-native-reanimated": "~2.14.4",
+    "react-native-reanimated": "2.17.0",
```

Comment:
Can you solve the conflicts with this file

Reply:
I think you'll need to commit a new yarn.lock as well

Reply:
I already resolved the conflicts and added the yarn.lock again